guidelines/decision-process.md

```diff
@@ -1 +1,23 @@
+# Decision making process
 
+## Preamble
+The Chair Group discussion led to a consensus to start with a light weight consensus-based process that would eventually use voting as a last resort method in case of impasse.. The group could eventually be in need to defend its decision making process against trolls, yet at the moment all group members are considered in good faith, so the working group reserves the option to specify disciplinary actions against trolls as part of its guidelines at a later stage..
```

Comment:
> so the working group reserves the option to specify disciplinary actions against trolls as part of its guidelines at a later stage..

 Not sure about the phrase(if it should belong to another place)

Reply:
This is an informal preamble explanation of why the blacklisting process is TBD atm

The whole preamble can be deleted, it doesn't add anything normative, it's just to explain why the rules are currently lightweight and partially underspecified.. 

It is intended to help the reader of the normative part to understand why things are the way they are ;-)

Reply:
I think it is foolish to leave this undefined--having experienced some world-class trolls and/or ill-behaved persons (you haven't lived until you have received death threats online??). We'd be better off to handle this [the way CLDR and ICU do](http://cldr.unicode.org/index/process): with a reference to the formal rules, located [here](http://www.unicode.org/consortium/tc-procedures.html). In practice, voting and such are almost never used in favor of consensus.

Reply:
> (you haven't lived until you have received death threats online??). We'd be better off t
😂😂😂😂😂😂😂😂😂


> We'd be better off to handle this [the way CLDR and ICU do](http://cldr.unicode.org/index/process): with a reference to the formal rules, located [here](http://www.unicode.org/consortium/tc-procedures.html). 

Agree!!